src/placement_lattice_plots.R

```diff
@@ -0,0 +1,416 @@
+library(dplyr)
+library(lubridate)
+library(tidyr)
+library(reshape2)
+library(ggthemes)
+library(tidyquant)
+source("src/helpers.R")
+
+generate_placement_lattice_plots <- function(input_dir, output_dir, historic_start, historic_end, projection_start, projection_end) {
```

Comment:
How come we don't infer most of these args from the input data?

Reply:
Just because there's potentially a lot more history / projection than what we want to show in the charts. There are often requests for specific date ranges, and this way we can re-use the same input data

Reply:
Great, thanks (I suspected as much!)